### ISSUE TITLE
Screaming invitation not getting highlighted

### ISSUE THREAD

**Issue by MarombeiroT** [veiset/poe-vendor-string#39]
I was using the highlight feature to craft screaming invitations and it doesn't get highlighted with this string "tal d|ege|e he|non|o s|d el|f bu|es of d".

**Comment by veiset** [comment]
Thanks for the report, but without more information its a bit hard to debug.

Which map did you try to highlight with that string?

**Comment by MarombeiroT** [comment]
As soon I got another one, I'll post the infos using ctrl+c ctrl+v.  
the "Screaming invitation" and "incandecent invitation" are "misc" maps, they can be crafted with almost the same mods present in maps. this maps contain a pinnacle boss fight. I'm currently farming both versions "screaming" and "incandecent", but i'm having problems only with the screaming one. The filter works fine at incadecent one.

https://www.poewiki.net/wiki/Screaming_Invitation

**Comment by veiset** [comment]
Thanks I will have to look closer at this. I will try to do some debugging this evening (in 6-7 hours).

I ran a quick check with adding the invitation names to the generating rule file. Adding the following map names to generator:

```
        "Screaming Invitation",
        "Incandescent Invitation",
        "Polaric Invitation",
        "Writhing Invitation",
```

This resulted in a conflict for Polaric invitaitons, but I didn't find anything for Screaming.

```
 "Area is inhabited by Solaris fanatics": {
   value: "Area is inhabited by Solaris fanatics",
   scary: 499,
   matchSafe: "laris",
 },
 ```
 
 Where `lari`  (conflicting with `PoLARIc` has been updated to `laris`.
 
But I think I found the issue, there are a few modifiers matching the flavor text. For instance the modifier `e he` (hexproof) matches `From thE HEart of the Tangle` in the flavor text from the screaming invitation. I will have a fix for this live on the page in around 7 hours.
